test/Event/ReadModel/Relations/ProjectorTest.php

```diff
@@ -282,4 +286,40 @@ public function it_stores_the_location_relation_when_the_place_of_an_event_is_up
 
         $this->projector->handle($domainMessage);
     }
+
+    /**
+     * @test
+     */
+    public function it_stores_the_location_relation_when_the_place_of_an_event_is_updated_via_a_major_info_update()
+    {
+        $eventId = 'event-id';
+        $locationId = 'location-id';
+        $majorInfoUpdatedEvent = new MajorInfoUpdated(
```

Comment:
FYI this is a deprecated event. We have more granular events for updates of these properties now. Sadly the front-end still uses an endpoint that results in this event (and there's historical events of this type in the event store).